core/lexicon/en/tv.inc.php

```diff
@@ -87,3 +86,16 @@
 $_lang['tv_sources.intro_msg'] = $_lang['tv_tab_sources_desc'];
 $_lang['tv_tmpl_access'] = $_lang['tv_tab_tmpl_access'];
 $_lang['tv_tmpl_access_msg'] = $_lang['tv_tab_tmpl_access_desc'];
+
+/*
+    Refer to default.inc.php for the keys below.
+    (Placement in this default file necessary to allow
+    quick create/edit panels access to them when opened
+    outside the context of their respective element types)
+
+    tv_tab_input_options
```

Comment:
By the way, don't forget about the comment :)

Reply:
I must be losing my mind! ;-) I made the change but managed to only push one of the three changed files. Anyway, you should see the change now.